xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -512,6 +516,70 @@ rebuild_master:
         configure_repositories step, set the <emphasis>add_repositories</emphasis> key to anything else than <emphasis>enabled</emphasis>.</para>
         </sect4>
 
+        <sect4 xml:id="sec.obs.obs_scm_ci_workflow_integration.obs_workflows.steps.submit_request">
+          <title>Submit a Request</title>
+
+          <para>The submit request step is the equivalent of the <emphasis>osc submitrequest</emphasis> command.</para>
+
+          <para>The requirements to run a submit request step are:</para>
+
+          <itemizedlist>
+            <listitem>
+              <para>There has to be a package branched already.</para>
```

Comment:
The package does not have to be a branch. It just needs to exist.

Reply:
I changed the two requirements. Please take another look.